presto-function-namespace-managers/pom.xml

```diff
@@ -149,6 +149,16 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.drift</groupId>
+            <artifactId>drift-transport-netty</artifactId>
```

Comment:
Do we need this dependency?  If so, why?

Reply:
I believe this dependency is needed for the MySqlFunctionNamespaceManagerFactory where it is creating a new DriftNettyClientModule(). I just changed the code position around because it was under the testing section so I put it above that section in the pom.xml file.